### DISCUSSION TITLE
質問についた回答の並び順について

### DISCUSSION THREAD

**Discussion by Verclene** [increments/qiita-discussions#492] [要望・アイデア・不具合]
一応検索はかけましたが既出でしたら申し訳ございません.

各「質問」についた回答は,「意見交換」を除きいいねの順に並んでいますが,これを時系列順に変更する方法はありますでしょうか.
質問者からの追記を回答欄でする方がたまにいて,流れが把握できなくなるケースが散発していて,少々読みにくいのです.
コメント欄へのMarkdown対応等でこうした問題は割と改善がみられましたが,回答の並び順についてもご助言,ご検討いただけましたら幸いです.

**Comment by tanaka-asami415** [comment]
@Verclene 
ご意見ありがとうございます。

ご不便をおかけして申し訳ありません。
現在は時系列順に表示する方法がないため、今後の質問改善の際に検討いたします🙏
